aip/general/0162.md

```diff
@@ -234,17 +149,12 @@ there are two potential variants:
 If a child resource is a child of a single revision, the child resource's name
 **must** always explicitly include the parent's revision ID:
 
-    publishers/123/books/les-miserables@c7cfa2a8/pages/42
-
-In `List` requests for such resources, the service **should** default to the
-latest revision of the parent if the user does not specify one, but **must**
-explicitly include the parent's revision ID in the `name` field of resources in
-the response.
+    publishers/123/books/les-miserables/revisions/c7cfa2a8/pages/42
 
 If necessary, APIs **may** explicitly support listing child resources across
-parent revisions by accepting the `@-` syntax. For example:
+parent revisions by accepting the `-` syntax. For example:
 
-    GET /v1/publishers/123/books/les-miserables@-/pages
+    GET /v1/publishers/123/books/les-miserables/revisions/-/pages
```

Comment:
Isn't this guidance redundant? the former is obvious: where do you put a child of a revision?

the latter is basically AIP-159. 

I feel like this part could just be removed.